examples/CDC/multiCDC/multiCDC.ino

```diff
@@ -0,0 +1,134 @@
+/*
+    This example demonstrates the use of multiple USB CDC/ACM "Virtual Serial" ports,
+    using Ha Thach's tinyUSB library, and the port of that library to the Arduino environment
+    https://github.com/hathach/tinyusb
+    https://github.com/adafruit/Adafruit_TinyUSB_Arduino
+
+    Written by Bill Westfield (aka WestfW), June 2021.
+    This code is released to the public domain (note that this is a different
```

Comment:
what is the different in the license of this example from the rest for the library ?

Reply:
That means I retain no rights at all to the example code.  Not copyrighted at all.
In particular, it should mean that you or Adafruit could remove my name and slap your own license on it.
I have no objection to using an MIT license instead, although I worry that many OSSW licenses get "complicated" as the number of contributors goes up.  (in this case, it's only in the example code.)

(I acknowledge the desire/need for a simpler example.  (though the example itself isn't that complex, other than the need to also include the parser library.))

Reply:
I would prefer to have an example as simple as possible just to demonstrate how the USB work. Please consider to change the example to echo all with just slight different in the texts. We could prefix it with "n>" (n is instance number). 

For the license, you could just have your name on top of the MIT license notice ( does need to be a full license, just make sure MIT is mentioned). We have no reason to change your name. If the example is modified significantly that person can insert their name next to yours.